packages/mongo-nav/src/mongo-select/OrgSelect.tsx

```diff
@@ -408,24 +409,30 @@ function OrgSelect({
               </FocusableMenuItem>
             )}
 
-            <ul className={ulStyle}>
+            <div role="none" className={ulStyle}>
```

Comment:
Can we add `type="search"` to the search inputs please?

Reply:
yep, adding now.